ic-agent/src/agent/mod.rs

```diff
@@ -12,7 +12,7 @@ pub use agent_error::AgentError;
 pub use builder::AgentBuilder;
 #[doc(inline)]
 pub use ic_transport_types::{
-    signed, EnvelopeContent, RejectCode, RejectResponse, ReplyResponse, RequestStatusResponse,
+    signed, Envelope, EnvelopeContent, RejectCode, RejectResponse, ReplyResponse, RequestStatusResponse,
```

Comment:
```suggestion
    signed, Envelope, EnvelopeContent, RejectCode, RejectResponse, ReplyResponse,
    RequestStatusResponse,
```
rustfmt complains about this